MekHQ/src/mekhq/gui/utilities/MarkdownEditorPanel.java

```diff
@@ -0,0 +1,260 @@
+package mekhq.gui.utilities;
```

Comment:
Needs file header, class comment, public method comments.

Reply:
Added javadocs in b74eda9 and header in 7f7b871